Go 1.15: conversion from int to string yields a string of one rune

1. What version of GopherLua are you using? :  git tip https://github.com/yuin/gopher-lua/commit/dc2b0ca8b37e41ab0494be94fc6d8febbeb29d34
2. What version of Go are you using? :  Go 1.15 rc 1
3. What operating system and processor architecture are you using? : Fedora Linux Rawhide, x86_64 and armv7hl
4. What did you do? : Run go test
5. What did you expect to see? : Tests pass.
6. What did you see instead? :
```
Testing    in: /builddir/build/BUILD/gopher-lua-dc2b0ca8b37e41ab0494be94fc6d8febbeb29d34/_build/src
         PATH: /builddir/build/BUILD/gopher-lua-dc2b0ca8b37e41ab0494be94fc6d8febbeb29d34/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/gopher-lua-dc2b0ca8b37e41ab0494be94fc6d8febbeb29d34/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/yuin/gopher-lua/version=0 -X github.com/yuin/gopher-lua/version.commit=dc2b0ca8b37e41ab0494be94fc6d8febbeb29d34 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/yuin/gopher-lua
github.com/yuin/gopher-lua
# github.com/yuin/gopher-lua
./utils.go:35:22: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL	github.com/yuin/gopher-lua [build failed]
```

See https://github.com/golang/go/issues/32479

--- a/utils.go
+++ b/utils.go
@@ -32,7 +32,7 @@ func defaultFormat(v interface{}, f fmt.State, c rune) {
 	buf = append(buf, "%")
 	for i := 0; i < 128; i++ {
 		if f.Flag(i) {
-			buf = append(buf, string(i))
+			buf = append(buf, string(rune(i)))
 		}
 	}
 

